grails-app/controllers/org/pih/warehouse/api/CycleCountApiController.groovy

```diff
@@ -182,7 +182,8 @@ class CycleCountApiController {
     }
 
     def refreshCycleCount(String cycleCountId) {
-        CycleCountDto cycleCount = cycleCountService.refreshCycleCount(cycleCountId)
+        boolean removeOutOfStockItemsImplicitly = params.boolean("removeOutOfStockItemsImplicitly", false)
```

Comment:
Nitpicky: What does "implicitly" mean in this context? Do we need it in this param?

Reply:
I wanted to somehow indicate that we „force” remove to happen - currently we might remove an item without this param, if the quantityCounted is <= 0, so I wanted it to sound like „remove an item, no matter how many other conditions we have”